darwin/importer/formats/labelbox.py

```diff
@@ -13,7 +15,7 @@
 )
 
 
-def parse_file(path: Path) -> Optional[List[AnnotationFile]]:
+def parse_file(path: Path, schema: Dict[str, Any]) -> Optional[List[AnnotationFile]]:
```

Comment:
I think that it now misses `schema` in the docstring.

Reply:
Correct!

Reply:
Waiting for final approval before fixing the docs :D

Reply:
Fixed